library/src/main/java/com/github/paolorotolo/appintro/CustomFontCache.java

```diff
@@ -0,0 +1,35 @@
+package com.github.paolorotolo.appintro;
+
+import android.content.Context;
+import android.graphics.Typeface;
+
+import java.util.Hashtable;
+
+/**
+ * Created by ameykshirsagar on 23/07/16.
+ */
+public class CustomFontCache {
+    private static Hashtable<String, Typeface> fCache = new Hashtable<String, Typeface>();
+
+    public static Typeface get(String tfn, Context ctx) {
+        Typeface tf = fCache.get(tfn);
+        if (tf == null) {
+            //Log.i("font add","Adding new font -> "+tfn);
+            try {
+                tf = Typeface.createFromAsset(ctx.getAssets(), tfn);
+                if (tf != null) {
+                    fCache.put(tfn, tf);
+                }
+
+                return tf;
+            } catch (Exception e) {
+                return null;
```

Comment:
Add a logging statement here, just in case~
It does not need to throw an exception, just a warn is fine.